test-integration/helpers/scores.coffee

```diff
@@ -29,6 +27,10 @@ COMMON_ELEMENTS =
     css: '.cc-cell a.score'
   averageLabel:
     css: '.average-label span:last-child'
+  exportUrl:
+    css: '#downloadExport'
+  doneGenerating:
+    css: "#downloadExport[src$='.xlsx']"
   assignmentByType: (type) ->
```

Comment:
Neat, I didn't know the "ends-with" selector was supported by IE, but apparently has been since IE 7.